### DISCUSSION TITLE
pyMOR Community Meeting 2024-01-23

### DISCUSSION THREAD

**Discussion by HenKlei** [pymor/pymor#2244] [Announcements]
The next pyMOR Community Meeting will take place Tuesday, January 23, at 15:00 CET and 9:00 EST, online at:

Link: http://bbb.mpi-magdeburg.mpg.de/meeting/pymor
Password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [ ] post-release todos (@lbalicki)
- [ ] pyMOR strategy meeting (@sdrave)
- [ ] docs deployment (@sdrave)
- [ ] [PDESoft 24](https://pdesoft.org/) (@sdrave)
- [ ] Examples for QR investigations (@drittelhacker)

Moderator: @HenKlei

**Comment by lbalicki** [comment]
Notes:
- post-release todos:
  - @sdrave will be next release manager 
  - main developers will go through open issues and assign them
- pyMOR strategy meeting
  -  we will set up a date by the end of the week
- docs deployment
  - docs repo archived
  - now everything works on muenster cloud
  - there are scripts for cleaning up
- PDESoft 24
  - MOR is advertised as topic of conference
  - depending on where we are going with pyMOR might be good to present there
- Examples for QR investigations
  - Discussed some benchmark problems for systems-theory

Next meeting Wednesday Feburary 7th, Moderator: @sdrave
